clients/cobol-lsp-vscode-extension/src/__tests__/services/SettingsTest.spec.ts

```diff
@@ -34,9 +35,7 @@ jest.mock("vscode", () => {
           throw { code: "FileNotFound" };
         }),
       },
-      getWorkspaceFolder: () => {
-        uri: new Uri("/c:/my/workspace");
-      },
+      getWorkspaceFolder: () => {},
```

Comment:
The fact that tests seem to be still pasing after this change should be investigated.

Reply:
Settings service is using `vscode.workspace.workspaceFolders` and it doesn't call `getWorkspaceFolder` at all. Additionally, there is mistake in original code. It's missing parentheses around the object literal, so the function always returned `undefined` as the `uri` is label and not the object property in that case. 
I think it supposed to look like this:
```ts
getWorkspaceFolder: () => ({
  uri: new Uri("/c:/my/workspace");
}),
```